analyses/stackanalyses/sa.go

```diff
@@ -0,0 +1,186 @@
+package stackanalyses
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"mime/multipart"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/rs/zerolog/log"
+
+	"github.com/fabric8-analytics/cli-tools/analyses/pypi"
+	"github.com/fabric8-analytics/cli-tools/utils"
+	"github.com/jpillora/backoff"
+)
+
+// SARequestType is a argtype of RequestServer func
+type SARequestType struct {
+	UserID          string
+	Host            string
+	ThreeScaleToken string
+	ShellPath       string
+	RawManifestFile string
+	DepsTreePath    string
+}
+
+// SAPostResponseType is a argtype of RequestServer func
+type SAPostResponseType struct {
+	SubmittedAt string `json:"submitted_at,omitempty"`
+	Status      string `json:"status,omitempty"`
+	ID          string `json:"id,omitempty"`
+}
+
+// SAGetResponseType is a argtype of RequestServer func
+type SAGetResponseType struct {
+	AnalysedDeps    []interface{}          `json:"analyzed_dependencies"`
+	Ecosystem       string                 `json:"ecosystem"`
+	Recommendation  map[string]interface{} `json:"recommendation"`
+	LicenseAnalyses map[string]interface{} `json:"license_analysis"`
+}
+
+// ReadManifestResponse is arg type of readManifest func
+type ReadManifestResponse struct {
+	DepsTreePath     string `json:"manifest,omitempty"`
+	RawFileName      string `json:"file,omitempty"`
+	Eco              string `json:"ecosystem,omitempty"`
```

Comment:
Don't use abbreviations.
```suggestion
	Ecosystem              string `json:"ecosystem,omitempty"`
```

Reply:
Ack